crawler/__init__.py

```diff
@@ -22,20 +22,26 @@ def create_app(config_object: str = None) -> Flask:
     # setup logging
     logging.config.dictConfig(app.config["LOGGING"])
 
-    if app.config.get("ENABLE_CHERRYPICKER_ENDPOINTS", False):
-        from crawler.blueprints import cherrypicker_test_data
-
-        app.register_blueprint(cherrypicker_test_data.bp)
-
     if app.config.get("SCHEDULER_RUN", False):
         scheduler.init_app(app)
         scheduler.start()
 
+    setup_blueprints(app)
+
     @app.get("/health")
-    def health_check():
+    def _():
         if scheduler.get_job(SCHEDULER_JOB_ID_RUN_CRAWLER):
             return "Crawler is working", HTTPStatus.OK
 
         return "Crawler is not working correctly", HTTPStatus.INTERNAL_SERVER_ERROR
 
     return app
+
+
+def setup_blueprints(app):
+    if app.config.get("ENABLE_CHERRYPICKER_ENDPOINTS", False):
+        from crawler.blueprints.v1 import cherrypicker_test_data as cptd_v1
+
+        app.register_blueprint(cptd_v1.bp, url_prefix="/v1")
+        app.register_blueprint(cptd_v1.bp)  # Also serve v1 at the root of the host for now
```

Comment:
I think it's worth making the change in lighthouse now to prevent this TODO kicking around for too long.

Reply:
Will do ... I just need to make that update in another branch then this will be coming out immediately